### PR TITLE
Adding a new gdb command to load enclave symbols.

### PR DESCRIPTION
**Pull request by VardhanThigle**
sgx-add-symbol-file <ADDR_IN_ENCLAVE | auto> <ELF_PATH>
auto indicates address is to be read from rbx.

This fixes [#19](https://github.com/fortanix/rust-sgx/issues/19)

### CODE REVIEW

File: scripts/gdb.py [hunk 1]
```diff
@@ -174,6 +178,28 @@ def set_state(self, registers):
     for r, v in registers.items():
       gdb.execute("set ${} = {}".format(r,v))
 
+class Sgx_Add_Symbol_File (gdb.Command):
+  """
+  Given TCS address and ELF path, load enclave's symbols.
```
Comment (jethrogb) [inline scripts/gdb.py]: “Given any address in the enclave and ELF path...”

Comment (VardhanThigle) [reply scripts/gdb.py]: Thanks. Fixed in current push.

Comment (jethrogb) [reply scripts/gdb.py]: Still says TCS in a few more places

Comment (jethrogb) [reply scripts/gdb.py]: Also I think you can say "Specifying auto is equivalent to specifying `$rbx`"

Comment (VardhanThigle) [reply scripts/gdb.py]: Changed the description in current push. Also removed TCS from the command syntax.

File: scripts/gdb.py [hunk 2]
```diff
@@ -174,6 +178,28 @@ def set_state(self, registers):
     for r, v in registers.items():
       gdb.execute("set ${} = {}".format(r,v))
 
+class Sgx_Add_Symbol_File (gdb.Command):
+  """
+  Given any address in the enclave and ELF path, load enclave's symbols.
+  sgx-add-symbol-file <ADDR_IN_ENCLAVE | auto> <ELF_PATH>
+  Specifing address as auto is equivalent to specifying it in $rbx.
```
Comment (jethrogb) [inline scripts/gdb.py]: "specifying it in $rbx" doesn't make sense. You can say "specifying $rbx"

Comment (jethrogb) [reply scripts/gdb.py]: s/Specifing/Specifying/

Comment (VardhanThigle) [reply scripts/gdb.py]: Corrected in recent push.

Comment (VardhanThigle) [reply scripts/gdb.py]: Corrected in recent push.

**Pull request merged**